docs/markdown/Release-notes-for-0.40.0.md

```diff
@@ -119,7 +119,7 @@ qt5_dep = dependency('qt5', modules : 'core', method : 'qmake')
 ## Link whole contents of static libraries
 
 The default behavior of static libraries is to discard all symbols
-that are not not directly referenced. This may lead to exported
+that are not directly referenced. This may lead to exported
```

Comment:
Sometimes double negatives are intentional. usually they're just typos. I'm pretty sure I'm right but not absolutely confident.